experimental/Schemes/ProjectiveModules.jl

```diff
@@ -16,7 +16,7 @@ function annihilator(M::SubquoModule)
   return I
 end
 
-iszero(I::Ideal) = all(x->(iszero(x)), gens(I))
+iszero(I::Ideal) = all(x->(iszero(x)), gens(I))  # TODO: move this to AA
```

Comment:
Moved to AA in https://github.com/Nemocas/AbstractAlgebra.jl/pull/1494 but that has no yet been released.